site/content/posts/2022-01-18-idp-refresh-tls-ciphers-for-compliance.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: "Pinniped v0.13.0: Security Hardened Pinniped"
 slug: secure-tls-idp-refresh
-date: 2022-01-21
+date: 2022-03-02
```

Comment:
This updates the date of the blog post itself, which is not what I think @margocrawf was asking for (since it makes it unclear when the blog was written).

Reply:
reverted back to original date and added excerpts mentioning the new release